eip155/caip10.md

```diff
@@ -32,11 +32,11 @@ Note that a given address cannot be assumed to work on all current and future ne
 
 ## Syntax
 
-Ethereum addresses were, historically, case-insensitive and normalized to use
-all-lowercase letters (`abcdef`) like most hexadecimal numeric types.  With the
-ratification of [EIP-55][], however, a particular normalization of lowercase- and
-uppercase- `abcdefABCDEF` characters was invented as an efficient form of
-checksum. See [EIP-55][] for specification.
+Ethereum addresses were, historically, case-insensitive and normalized to use all-lowercase letters (`abcdef`) like most hexadecimal numeric types.
+With the ratification of [EIP-55][], however, a particular normalization of lowercase- and uppercase- `abcdefABCDEF` characters was invented as an efficient form of checksum.
```

Comment:
Isn't this ERC-55 now?

Reply:
yes? "historically" meaning before EIP-55.  do i misunderstand?